### ISSUE TITLE
Investigate Revue for newsletter hosting

### ISSUE THREAD

**Issue by laurenluz** [Giveth/giveth-planning#644]
https://www.getrevue.co/

**Comment by laurenluz** [comment]
yeah! i think this is cool. let's add a subdomain, maybe make it shorter like: news.giveth.io

do you need any support on that? I'd love to link it to our Twitter

**Comment by ahmadabugosh** [comment]
This is currently ready, but just waiting for the subdomain (news.giveth.io) to propagate (Kay already did it but it may need more time)

**Comment by ahmadabugosh** [comment]
Closing this issue and discussion moved to [](https://github.com/Giveth/giveth-planning/issues/669)
